docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -361,6 +361,19 @@ This value limits the size of a single cache value. If a cache value (or query r
 
 The default is `1`.
 
+## [caching.encryption]
+### enabled
+
+Setting 'enabled' to `true` allows users to encrypt the query values in the cache.
+
+This value is `false` by default.
+
+### encryption_key
+
+The string used to generate a key for encrypting the cache. In order for encrypted cache data to persist between Grafana restarts, the key must be specified. If left empty when encryption is enabled, the key will be automatically generated on startup and the cache will be cleared upon restarts.
```

Comment:
```suggestion
This string generates a key for encrypting the cache. For the encrypted cache data to persist between Grafana restarts, this key must be specified. If it is left empty when encryption is enabled, then the key is automatically generated on startup and the cache is cleared upon restarts.
```

Reply:
Updated! Sorry for the delayed update - I was out all last week.